eslint.config.js

```
@@ -0,0 +1,35 @@
+const js = require('@eslint/js')
```

ESLint

[error] 'require' is not defined.

(no-undef)